internal/converter/testdata/maps.go

```diff
@@ -6,19 +6,44 @@ const Maps = `{
     "definitions": {
         "Maps": {
             "properties": {
-                "map_of_strings": {
+                "map_of_strings_to_strings": {
+                    "mapKey": "string",
                     "additionalProperties": {
                         "type": "string"
                     },
                     "type": "object"
                 },
-                "map_of_ints": {
+                "map_of_strings_to_ints": {
+                    "mapKey": "string",
                     "additionalProperties": {
                         "type": "integer"
                     },
                     "type": "object"
                 },
-                "map_of_messages": {
+                "map_of_strings_to_messages": {
+                    "mapKey": "string",
+                    "additionalProperties": {
+                        "$ref": "#/definitions/samples.PayloadMessage",
+                        "additionalProperties": true
+                    },
+                    "type": "object"
+                },
+                "map_of_ints_to_strings": {
+                    "mapKey": "integer",
+                    "additionalProperties": {
+                        "type": "string"
+                    },
```

Comment:
can we just make this `valueType` and not nest it under `additionalProperties`?

Reply:
This seems like a larger change to the schema? I intentionally didnt touch this pathway, seems best to conform as closely as before

Reply:
oh nvm, this was already there. disregard! thanks.